workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -336,6 +336,10 @@ pre[class*='language-'] {
 			color: var(--color-neutral-8, #54555f);
 		}
 
+		.learn-card:has(.subsection a) h4 {
```

Comment:
@luisacmn we have a dedicated section to .learn-card lower in this file that styles the h4s, are we able to add these styles there to keep things cleaner?

Reply:
@ryanschuhler sure ! thanks for the suggestion
I just updated the PR